Rakefile

```diff
@@ -0,0 +1,96 @@
+require 'json'
+require 'faraday'
+require 'fileutils'
+require 'logger'
+require 'rubygems'
+require 'rspec/core/rake_task'
+
+RSpec::Core::RakeTask.new(:spec)
```

Comment:
Have you tested this in a heroku deploy?  Given that the `rspec` gem is in the `:test` group, I thought that this usually resulted in `rspec/core/rake_task` being unavailable during heroku slug compilation, requiring some `rescue LoadError` and `if defined?(RSpec)`.